util.js

```diff
@@ -640,4 +640,20 @@
         return url && url.replace(/^https/, "http");
     };
 
+    /**
+     * Unescapes HTML entities so the text in which they appear can be displayed as text of DOM elements
+     * 
+     * This implementation is as suggested by the MDN:
+     * https://developer.mozilla.org/en-US/Add-ons/Code_snippets/HTML_to_DOM
+     *
+     * @param {String} html foto escape entities
+     * 
+     * @return {String}
+     */
+    DDG.unescape = function(text) {
+        var doc = document.implementation.createHTMLDocument("example");
```

Comment:
Wow, I've never seen this done before. It seems to be supported by [most browsers apart from <IE9](https://developer.mozilla.org/en-US/docs/Web/API/DOMImplementation/createHTMLDocument#Browser_compatibility), but as long as we don't use it on static pages it should be fine.

The other thing I'm worried about is that creating a document object just to unescape some some strings might be really slow. I'll play around with it for a bit and see if that's the case.